src/sentry/integrations/message_builder.py

```diff
@@ -24,15 +24,36 @@ class AbstractMessageBuilder(ABC):
     pass
 
 
-def format_actor_options(actors: Sequence[Team | RpcUser]) -> Sequence[Mapping[str, str]]:
+def format_actor_options(
+    actors: Sequence[Team | RpcUser], use_block_kit: bool = False
+) -> Sequence[Mapping[str, str]]:
     sort_func: Callable[[Mapping[str, str]], Any] = lambda actor: actor["text"]
-    return sorted((format_actor_option(actor) for actor in actors), key=sort_func)
+    if use_block_kit:
+        sort_func = lambda actor: actor["text"]["text"]
```

Comment:
is `["text"]["text"]` guaranteed to be there?

Reply:
Yeah if you look at `format_actor_option` just below it, it returns the nested text.